packages/dx-react-chart/src/plugins/axis.jsx

```diff
@@ -136,7 +143,7 @@ export class Axis extends React.Component {
                       position: 'absolute', left: 0, top: 0, overflow: 'visible',
                     }}
                   >
-                    <Root
+                    <RootComponent
```

Comment:
Usually, we prefer don't use *Component postfix for components.

Reply:
`Root` is already occupied in the outer scope.